front/src/shared.module.scss

```
@@ -0,0 +1,168 @@
+@import './colors.module.scss';
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #d6d1bd;
+  /* enable in development
+     background-color: pink; */
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+.layout-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.layout-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.arrow {
+  @extend .layout-row;
+  align-items: center;
+  width: 0;
+  height: 0;
+  border-top: 20px solid transparent;
+  border-bottom: 20px solid transparent;
+  cursor: pointer;
+}
+
+.arrow-right {
+  @extend .arrow;
+  margin-left: 5vw;
+  float: right;
+  border-left: 20px solid $COLOR_BLACK;
+}
+.arrow-left {
+  @extend .arrow;
+  margin-right: 5vw;
+  text-align: right;
+  border-right: 20px solid $COLOR_BLACK;
+}
+
+// FROM INFOBOXES
+
+.infoItem {
+    display: flex;
+    min-width: 320px;
+    width: 30vw;
+    height: 38px;
+    justify-content: left;
+}
+
+.box {
+    float: left;
+    height: 20px;
+    width: 20px;
+    margin-right: 8px;
+    margin-left: 0px;
+    margin-bottom: 0px;
+    border: 1px $COLOR_BLACK;
+    clear: both;
+    min-height: 20px;
+    min-width: 20px;
+}
+
+.noFlex {
+    display: inline-block;
+}
+
+.infoItemImg {
+    position: relative;
+    display: flex;
+    min-width: 320px;
+    width: 30vw;
+    height: 38px;
+    justify-content: left;
+}
+
+.infoText {
+    display: inline;
+    overflow: auto;
+    margin-bottom: 0;
+}
+
+.relativeText {
+    position: absolute;
+    bottom: 0;
+    left: 30px;
+}
+
+// FROM MONTHVIEW
+
+.viewChangerCurrent {
+  display: flex;
+  justify-content: flex-start;
+  border-right: 1px solid;
+}
+
+.viewChangerCurrent > a {
+  padding: 8px;
+  padding-top: 16px;
+  align-items: center;
+  border: 1px;
+  border-bottom: 0;
+  border-style: solid;
+  border-color: $COLOR_BLACK;
+  font-size: large;
+  font-weight: bold;
+  background-color: $COLOR_LIGHT_GREY;
+  color: $COLOR_LIGHT_GREY;
+}
+
+.viewChangerCurrent > a > div {
+  color: $COLOR_BLACK;
+}
+
+.viewChanger {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 57px;
+  font-weight: bold;
+}
+
+.viewChangerContainer {
+  display: flex;
+  justify-content: flex-end;
+  border-right: 1px solid;
+}
+
+.viewChangerContainer > a {
+  padding: 8px;
+  padding-top: 16px;
+  align-items: center;
+  border: 1px;
+  border-right: 0px;
+  border-bottom: 0;
+  border-style: solid;
+  border-color: $COLOR_BLACK;
+  font-size: large;
+  font-weight: bold;
+  background-color: $COLOR_LIGHT_GREY;
+  color: $COLOR_GREY;
+}
+
+.viewChangerContainer > a > div {
+  color: $COLOR_BLACK;
+}
```

The style repetition of .viewChangerCurrent > a and .viewChangerContainer > a could be reduced here
```suggestion
.viewChangerLink {
  padding: 8px;
  padding-top: 16px;
  align-items: center;
  border: 1px;
  border-style: solid;
  border-color: $COLOR_BLACK;
  font-size: large;
  font-weight: bold;
  background-color: $COLOR_LIGHT_GREY;
}

.viewChangerCurrent > a {
  @extend .viewChangerLink;
  color: $COLOR_LIGHT_GREY;

  div {
    color: $COLOR_BLACK;
  }
}

.viewChanger {
  display: flex;
  justify-content: space-between;
  width: 100%;
  height: 57px;
  font-weight: bold;
}

.viewChangerContainer {
  display: flex;
  justify-content: flex-end;
  border-right: 1px solid;
}

.viewChangerContainer > a {
  @extend .viewChangerLink;
  color: $COLOR_GREY;

  div {
    color: $COLOR_BLACK;
  }
}
```